node/cn/tracers/api.go

```diff
@@ -40,11 +41,12 @@ import (
 	"github.com/klaytn/klaytn/blockchain/vm"
 	"github.com/klaytn/klaytn/common"
 	"github.com/klaytn/klaytn/common/hexutil"
+	"github.com/klaytn/klaytn/consensus"
 	"github.com/klaytn/klaytn/log"
 	"github.com/klaytn/klaytn/networks/rpc"
-	"github.com/klaytn/klaytn/node/cn/tracers"
+	"github.com/klaytn/klaytn/params"
 	"github.com/klaytn/klaytn/rlp"
-	statedb2 "github.com/klaytn/klaytn/storage/statedb"
```

Comment:
Do we need to change the name `statedb2` here?

Reply:
It is changed from `statedb` package to `database` package here. So I think it is not necessary to use the name `statedb2` here. If i miss something, please tell me..